app/assets/stylesheets/buttons.scss

```diff
@@ -0,0 +1,42 @@
+@import 'text';
+
+.btn {
+    @include subtitle($color: false);
+    /* min-width: 167px; */
+}
+
+.primaryButton{
+    font-family: Roboto;
+    font-style: normal;
+    font-weight: 700;
+    letter-spacing: 0.08em;
+    font-size: 18px;
+    line-height: 21px;
+    text-align: center;
+    color: white;
+    background-color: $Primary06;
+    border-radius: 5px;
+    border-color: transparent;
+    padding-top: 5px;
+    padding-bottom: 5px;
+    padding-left: 11px;
+    padding-right: 11px;
+}
+
+.secondaryButton{
+    font-family: Roboto;
+    font-style: normal;
+    font-weight: 700;
+    letter-spacing: 0.08em;
+    font-size: 18px;
+    line-height: 21px;
+    text-align: center;
+    color: white;
+    background-color: $Grey10;
+    border-radius: 5px;
+    border-color: transparent;
+    padding-top: 5px;
+    padding-bottom: 5px;
+    padding-left: 11px;
+    padding-right: 11px;
+}
```

Comment:
Have you thought about following the naming conventions of bootstrap? 
I would highly appreciate if we could follow these since this would greatly simplify migrations for existing pages. 

See this for docs: https://getbootstrap.com/docs/5.0/components/buttons/

Reply:
These names originate from the figma design. We thought it would be good to keep a close relationship to our figma as this should be the place most people look when picking a design.
Besides that we weren't totally sure if bootstrap was supposed to be a bigger part of our design concept. As far as I know (and i can be very wrong on that) people just started using bootstrap out of convenience and without any bigger, unified design in mind.